### PR TITLE
LCH-6074: Implement general command.

### PR DESCRIPTION
**Pull request by saxumVermes**

### CODE REVIEW

File: src/Command/Helpers/VersionFetcher.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  /**
+   * Drupal.org API update url.
+   */
+  public const UPDATES_URL = 'https://updates.drupal.org';
+
+  /**
+   * Returns the latest supported version of the project.
+   *
+   * Security covered releases are considered stable.
+   *
```
Comment (narendradesai) [inline src/Command/Helpers/VersionFetcher.php:21]: Please add `@param` hint for `$projectName` in doc.

**Review by narendradesai — CHANGES_REQUESTED** [review]

**Review by amangrover90 — COMMENTED** [review]
Overall approach looks good to me, just a small comment about service reusability.

**Review by narendradesai — APPROVED** [review]
LGTM 👍🏻

**Review by amangrover90 — APPROVED** [review]
LGTM!!

**Pull request merged**